doc/build/dependency.md

```diff
@@ -0,0 +1,94 @@
+# Building dependencies
```

Comment:
Yes, I updated the version of the NDK and all the other build tools from Android Studio. The version of NDK I have currently installed is 14.1.3816874 and is the only one (I don't have other versions installed with brew). When I try to build a dependency using the suggestions provided in the documentation I get an error because it expects the NDK_DIR and not the NDK build script.

i.e. on my system

    $ ./build/android/dependency ~/Library/Android/sdk/ndk-bundle/ndk-build libevent 

should be 

    $ ./build/android/dependency ~/Library/Android/sdk/ndk-bundle/ libevent

Can you check if the previous command works in your system?